MetaMask/metamask-extension
# UX: Multichain: Send Flow: Hex Data

## **Description**

Adds the "Hex Data" label and content for the Send flow.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1453

## **Manual testing steps**

1.  Enter the new Send flow
2. Select a recipient
3. See hex data field

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

<img width="421" alt="SCR-20231031-ocdj" src="https://github.com/MetaMask/metamask-extension/assets/46655/5ddfe6ce-be48-4de6-a6ed-d565e675de26">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
